src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -244,8 +256,11 @@ public Event messageID(MessageID attachedMessageID) {
      * @deprecated prefer to use full object instead of part of them, use the {@link #toListProto(EventID)} method
      */
     @Deprecated
-    public List<com.exactpro.th2.common.grpc.Event> toProtoEvents(@Nullable String parentID) throws JsonProcessingException {
-        return toListProto(toEventID(parentID));
+    public List<com.exactpro.th2.common.grpc.Event> toProtoEvents(
```

Comment:
This is part of a public API and I'm not sure about increasing the major version. Let's create a separate override method to pass additional parametres

Reply:
Discussed in chat